helfi_features/helfi_media/config/override/oembed_providers.provider.icareus_suite.yml

```diff
@@ -0,0 +1,12 @@
+endpoints:
+  -
+    schemes:
+      - 'https://www.helsinkikanava.fi/*/web/helsinkikanava/player/embed/vod?assetId=*'
+      - 'http://www.helsinkikanava.fi/*/web/helsinkikanava/player/embed/vod?assetId=*'
```

Comment:
I don't think `http` urls should be allowed since the content served over http won't be loaded when using https.

Reply:
Good point, change done!